packages/nocodb/src/lib/db/sql-client/lib/mssql/MssqlClient.ts

```diff
@@ -2144,6 +2143,132 @@ class MssqlClient extends KnexClient {
     return result;
   }
 
+  /**
+   *
+   * @param {Object} - args
+   * @param {String} - args.parentTable
+   * @param {String} - args.parentColumn
+   * @param {String} - args.childColumn
+   * @param {String} - args.childTable
+   * @returns {Promise<{upStatement, downStatement}>}
+   */
+   async relationCreate(args) {
```

Comment:
Let me chime in real quick.

These JSDocs are incorrect as shown by Webstorm:
<img width="545" alt="image" src="https://user-images.githubusercontent.com/78412429/200110413-b433acdf-3e22-4442-a545-d96f7c5330ee.png">

`Parameter args is not described in JSDoc `

And it's kind of unnecessary to use JSDocs for typing when we're already using TypeScript that handles that.
It even limits the typing because you describe return values as `Object` when in reality they aren't just `Object` but something else, for example `Result`.

Imo: Remove the JSDocs comments.

Reply:
Thank you for comment. I agree those feel like duplicates with ts types.
I derived JSDocs from KnexClient. As those are mostly implemented by Pranav, @pranavxc what do you think on this?

Note: even if we decide removing/updating them, it is better we create a separate chore PR and get this one in as this PR is already big enough

Reply:
We added them when the entire project was in Javascript, since we migrated to Typescript we can remove them. We can do the cleanup part in a separate PR.